### PR TITLE
DM-35474: write the positionError field of the actuators telemetry topic, if present

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: python/lsst/ts/mthexapod/hexapod_csc.py [hunk 1]
```diff
@@ -547,6 +556,7 @@ async def do_move(self, data):
         as it allows running with invalid compensation coefficients or inputs.
         """
         self.assert_enabled()
+        self.move_event.set()
```
Comment (teweitsai) [inline python/lsst/ts/mthexapod/hexapod_csc.py:559]: Should we set the event after knowing the low-level controller accepts this command? Under `self._move()`?

File: python/lsst/ts/mthexapod/hexapod_csc.py [hunk 2]
```diff
@@ -566,6 +576,7 @@ async def do_offset(self, data):
         See note for do_move regarding checking the target position.
         """
         self.assert_enabled()
+        self.move_event.set()
```
Comment (teweitsai) [inline python/lsst/ts/mthexapod/hexapod_csc.py:579]: Should we set the event after knowing the low-level controller accepts this command? Under self._move()?

**Review by teweitsai — APPROVED** [review]
Looks good. Thanks for this update!

File: python/lsst/ts/mthexapod/hexapod_csc.py (hunk 1 above)
Comment (r-owen) [reply python/lsst/ts/mthexapod/hexapod_csc.py:559]: This event merely indicates that a move command has been received. I'll rename it to add clarity.

**Pull request merged**